core/src/main/kotlin/com/malinskiy/marathon/cache/CacheService.kt

```diff
@@ -0,0 +1,7 @@
+package com.malinskiy.marathon.cache
+
+interface CacheService {
+    suspend fun load(key: CacheKey, reader: CacheEntryReader): Boolean
+    suspend fun store(key: CacheKey, writer: CacheEntryWriter)
```

Comment:
Should it also be boolean? Maybe we won't be able to write something in the directory

Reply:
I mirrored Gradle build cache interfaces (see `BuildCacheService.java`) and I think we don't need boolean value here as we don't care about the result